profiles/common/modules/features/multisite_path_override/multisite_path_override.module

```diff
@@ -2,49 +2,40 @@
 /**
  * @file
  * Code for the multisite_path_override feature.
- *
- * This feature allows
  */
 
-//Modify pathauto settings form to add customisation options.
+/**
+ * Modify pathauto settings form to add customisation options.
+ */
 function multisite_path_override_form_pathauto_settings_form_alter(&$form, &$form_state) {
   $punctuation = pathauto_punctuation_chars();
-
   $weight = 0;
   foreach ($punctuation as $name => $details) {
-    //Add an option for the replacement. Add field to save the string.
+    // Add an option for the replacement. Add field to save the string.
     $form['punctuation']['pathauto_punctuation_' . $name]['#options'][] = t('Replace with');
     $form['punctuation']['pathauto_punctuation_' . $name]['#weight'] = $weight;
     $form['punctuation']['pathauto_punctuation_' . $name . '_val'] = array(
-        '#title' => t('The string you want to show instead of ' . $name . ':'),
-        '#type' => 'textfield',
-        '#weight' => $weight+1,
-        '#states' => array(
-          'visible' => array(
-            ':input[name="pathauto_punctuation_' . $name . '"]' => array('value' => '3'),
-          ),
+      '#title' => t('The string you want to show instead of @name:', array('@name' => $name)),
+      '#type' => 'textfield',
+      '#weight' => $weight + 1,
+      '#states' => array(
+        'visible' => array(
+          ':input[name="pathauto_punctuation_' . $name . '"]' => array('value' => '3'),
         ),
-        '#default_value' => variable_get('pathauto_punctuation_' . $name . '_val', ''),
-        //TODO: Add validation
-        /*'#rules' => array(
-          array(
-              'rule' => 'alpha',
-              'error' => 'Please, use only alphabetic characters.',
-          )
-        ),*/
+      ),
+      '#default_value' => variable_get('pathauto_punctuation_' . $name . '_val', ''),
+      // TODO: Add validation.
     );
     $weight++;
   }
 
- return system_settings_form($form);
+  return system_settings_form($form);
 }
 
-/*
+/**
  * Implements hook_pathauto_alias_alter().
  */
 function multisite_path_override_pathauto_alias_alter(&$alias, array &$context) {
   // Replace punctuation with string value.
-
-  $alias = str_replace("-","moins", $alias);
+  $alias = str_replace("-", "moins", $alias);
```

Comment:
?!?? Out of scope for this ticket, but this is very strange. This cannot ever habe been the intention.